src/tools/interop/interop.cpp

```diff
@@ -647,6 +662,17 @@ class InteropConnection {
                 QuicEventSet(pThis->QuackAckReceived);
             }
             break;
+        case QUIC_CONNECTION_EVENT_RESUMPTION_TICKET_RECEIVED:
+            if (Event->RESUMPTION_TICKET_RECEIVED.ResumptionTicketLength) {
+                pThis->ResumptionTicketLength = Event->RESUMPTION_TICKET_RECEIVED.ResumptionTicketLength;
+                pThis->ResumptionTicket = new uint8_t[pThis->ResumptionTicketLength];
+                memcpy(
+                    (uint8_t*)pThis->ResumptionTicket,
+                    Event->RESUMPTION_TICKET_RECEIVED.ResumptionTicket,
+                    Event->RESUMPTION_TICKET_RECEIVED.ResumptionTicketLength);
+            }
+            QuicEventSet(pThis->TicketReceived);
```

Comment:
Should this only be set in the case where the resumption ticket length is non-zero? The benefit is it might help us catch implementation bugs, but it'll look like the test timed out.

Reply:
Why would a nonzero length ever be received? I don't think you should special case this.